### PR TITLE
GIT Workshop intro slides and Event page updates

### PR DESCRIPTION
**Pull request by kldickenson**
These intro slides include TDA, the Sunday food sponsor.

### CODE REVIEW

File: events.html [hunk 1]
```diff
@@ -49,6 +49,7 @@
 
                     <h3>March 2016</h3>
                     <ul>
+                        <li>3/12/16 - <a href="events/0316_git.html">Intro to Version Control</a></li>
```
Comment (jewlofthelotus) [inline events.html]: let's use the full event title here

File: events/0316_git.html [hunk 2]
```diff
@@ -0,0 +1,215 @@
...
+    <div class="reveal">
+        <!-- Any section element inside of this container is displayed as a slide -->
+        <div class="slides">
+
+            <!-- Title -->
+            <section>
+                <img src="images/gdi_logo_badge.png" class="img--bare" height="450px" />
+                <div class="box--small">
+                    <h3 class="event__title">Intro to Version Control<br/><small><span class="green">Git, GitHub, &amp; the Terminal</span></small></h3>
+                    <p><small>James York &nbsp; | &nbsp; @kroysemaj &nbsp; | &nbsp; @gdiannarbor &nbsp; | &nbsp; #GDIA2 &nbsp; | &nbsp; #git-intro</small></p>
```
Comment (jewlofthelotus) [inline events/0316_git.html]: I've started removing event specific hashtags b/c I think they deter from the main #GDIA2 tag, so let's get rid of #git-intro wherever it exists

File: events/0316_git.html [hunk 3]
```diff
@@ -0,0 +1,215 @@
...
+                <h3>Workshop Slides</h3>
+                <div class="box"><a target="_blank" href="http://bit.ly/#">bit.ly/#</a></div>
+            </section>
+
+            <!-- Link to Slides -->
+            <section>
+                <h3>Slack!</h3>
+                <div class="box"><a target="_blank" href="https://gdiaa.slack.com/messages/0316-git-workshop/team/">#0316-git-workshop</a></div>
+                <p><small>Not on Slack? <a target="_blank" href="http://bit.ly/gdiaa-slack">bit.ly/gdiaa-slack</a></small></p>
+            </section>
```
Comment (jewlofthelotus) [inline events/0316_git.html]: let's move the workshop link and slack info slides to just before the closing slide, this just allows us to leave them up for a little longer than we'd be able to with them in the beginning here

File: events/0316_git.html [hunk 4]
```diff
@@ -0,0 +1,215 @@
...
+                    <h3 class="event__title">Meet James</h3>
+                    <img src="images/james.jpeg" class="img--bare box" width="300" />
+                    <p><small>James York &nbsp; | &nbsp; @kroysemaj &nbsp; | &nbsp; @gdiannarbor &nbsp; | &nbsp; #GDIA2 &nbsp; | &nbsp; #git-intro</small></p>
+                </div>
+            </section>
+
+            <!-- Introduce Instructor -->
+            <section>
+                <div class="box--small">
+                    <h3 class="event__title">Thanks Karen!!</h3>
```
Comment (jewlofthelotus) [inline events/0316_git.html]: feel free to add your head shot and social link here! :)

File: events/0316_git.html [hunk 5]
```diff
@@ -0,0 +1,215 @@
...
+                </div>
+            </section>
+
+            <!-- Link to Slides -->
+            <section>
+                <h3>Workshop Slides</h3>
+                <div class="box"><a target="_blank" href="http://bit.ly/#">bit.ly/#</a></div>
+            </section>
```
Comment (jewlofthelotus) [inline events/0316_git.html]: we can get rid of this slide after moving the two up above down here.

File: events/0316_git.html [hunk 6]
```diff
@@ -0,0 +1,215 @@
...
+                    <li><strong><span>Ladies That UX Happy Hour</span></strong><br /><small>Tuesday 3/22 @ Bobcat Bonnies, Detroit</small></li>
+                    <li><strong>VoiceOps; Command &amp; Control your AWS Environements</strong><br /><small>Thursday, 3/17 @ Duo Security</small></li>
+                    <li><strong><span>Web Analytics Wednesday</span></strong><br /><small>Wednesday, 3/23 @ AADL Downtown</small></li>
+                    <li><strong><span>Coffee House Coders</span></strong><br /><small>Wednesday, 3/23 @ Espresso Royal - State St.</small></li>
+                    <li><strong><span class="blue">self.conference</span></strong><br /><small>Friday, 5/20 and Saturday, 5/21 @ Motor City Casino<br/><span class="red">Use code "gdi-discount" for 15% off ticket price!</span></small></li>
+                    <li class="green"><em>Plus soooo many more...</em></li>
+                </ul>
+                <p class="clear copy--small"><a href="http://bit.ly/gdiaa-community">bit.ly/gdiaa-community</a></p>
+                <aside class="notes"></aside>
+            </section>
```
Comment (jewlofthelotus) [inline events/0316_git.html]: Can you make sure the group name is present before the event title - I think some of these are missing it. But this just makes it easier for people to find these events after they leave.

**Comment by kldickenson** [comment]
All these changes have been incorporated!
For Grand Circus events I left them as is since the location incorporates their name.

File: events/0316_git.html [hunk 7]
```diff
@@ -0,0 +1,211 @@
...
+                    <img src="images/james.jpeg" class="img--bare box" width="300" />
+                    <p><small>James York &nbsp; | &nbsp; @kroysemaj &nbsp; | &nbsp; @gdiannarbor &nbsp; | &nbsp; #GDIA2</small></p>
+                </div>
+            </section>
+
+            <!-- Introduce Instructor -->
+            <section>
+                <div class="box--small">
+                    <h3 class="event__title">Thanks Karen!!</h3>
+                    <img src="images/karen-headshot-2016.jpeg" class="img--bare box" width="250" />
```
Comment (jewlofthelotus) [inline events/0316_git.html]: Looks like you need to commit this image

File: events/0316_git.html [hunk 8]
```diff
@@ -0,0 +1,211 @@
...
+                    <p><small>Karen Dickenson &nbsp; | &nbsp; @kldickenson &nbsp; | &nbsp; @gdiannarbor &nbsp; | &nbsp; #GDIA2</small></p>
+                </div>
+            </section>
+
+            <!-- Link to Slides -->
+            <section>
+                <h3>Workshop Slides</h3>
+                <div class="box"><a target="_blank" href="http://bit.ly/#">bit.ly/#</a></div>
```
Comment (jewlofthelotus) [inline events/0316_git.html]: The slide URL needs to be set

**Comment by kldickenson** [comment]
Now the events page is updated with the full class name.

**Pull request merged**